WordPress/Classes/ViewRelated/NUX/NUXButtonViewController.swift

```diff
@@ -32,15 +38,23 @@ class NUXButtonViewController: UIViewController {
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
 
-        bottomButton?.setTitle(bottomButtonTitle, for: UIControlState())
-        bottomButton?.accessibilityIdentifier = accessibilityIdentifier(for: bottomButtonTitle)
-
-        topButton?.setTitle(topButtonTitle, for: UIControlState())
-        topButton?.accessibilityIdentifier = accessibilityIdentifier(for: topButtonTitle)
+        if let buttonConfig = bottomButtonConfig, let bottomButton = bottomButton {
+            bottomButton.setTitle(buttonConfig.title, for: UIControlState())
+            bottomButton.accessibilityIdentifier = accessibilityIdentifier(for: buttonConfig.title)
+            bottomButton.isPrimary = buttonConfig.isPrimary
+            bottomButton.isHidden = false
+        } else {
+            bottomButton?.isHidden = true
+        }
 
-        // Hide secondary button if title is not provided.
-        topButton?.isHidden = (topButtonTitle ?? "").isEmpty
-        bottomButton?.isHidden = (bottomButtonTitle ?? "").isEmpty
+        if let buttonConfig = topButtonConfig, let topButton = topButton {
+            topButton.setTitle(buttonConfig.title, for: UIControlState())
+            topButton.accessibilityIdentifier = accessibilityIdentifier(for: buttonConfig.title)
+            topButton.isPrimary = buttonConfig.isPrimary
+            topButton.isHidden = false
+        } else {
+            topButton?.isHidden = true
+        }
```

Comment:
Since these `if` statements are exactly the same, what do you think about making a single method to do the config? Something like:
```    
override func viewWillAppear(_ animated: Bool) {
        super.viewWillAppear(animated)

        configure(button: bottomButton, withConfig: bottomButtonConfig)
        configure(button: topButton, withConfig: topButtonConfig)
    }

    private func configure(button: NUXButton?, withConfig buttonConfig: NUXButtonConfig?) {
        if let buttonConfig = buttonConfig, let button = button {
            button.setTitle(buttonConfig.title, for: UIControlState())
            button.accessibilityIdentifier = accessibilityIdentifier(for: buttonConfig.title)
            button.isPrimary = buttonConfig.isPrimary
            button.isHidden = false
        } else {
            button?.isHidden = true
        }
    }
```